src/site/xdoc/server/config-blobstore.xml

```diff
@@ -56,6 +56,14 @@
                 </dl>
             </subsection>
 
+            <subsection name="Hybrid BlobStore option to always write to ObjectStorage">
+                <dl>
+                    <dt><strong>hybrid.duplicate.writes</strong></dt>
+                    <dd>DEFAULT: false, must be true/false. It happen only 'value = true' and it stored in cassandra.</dd>
```

Comment:
My suggestion:
"DEFAULT: false, must be true/false. This option is activated when 'value = true' and the blob will be stored in Cassandra"